routing/router.go

```diff
@@ -1818,6 +1820,12 @@ func (r *ChannelRouter) sendPayment(payment *LightningPayment,
 				update *lnwire.ChannelUpdate,
 				pubKey *btcec.PublicKey) {
 
+				err := ValidateChannelUpdateAnn(pubKey, update)
+				if err != nil {
+					log.Errorf("Unable to validate channel update: %v", err)
```

Comment:
This line and others too exceed the max line width of 80 chars (project coding standard).

Reply:
The log line is of 77 width. I believe I kept the <80 width everywhere.

Reply:
I have it at 96 chars. Do you have tab size 8?

Reply:
The standard is 80 chars max @ tab size 8.